Demo imageprovider docs (#4273)
open-telemetry/opentelemetry.io: content/en/docs/demo/services/imageprovider.md

---

> title: Imageprovider Service

Reviewer: ```suggestion
title: Image Provider Service
```

> linkTitle: Imageprovider

Reviewer: ```suggestion
linkTitle: Image Provider
```

> aliases: [imageprovider]

Reviewer: ```suggestion
```

---

> This service provides the images whcih are used in the frontend. Thei mages are statically hosted on a nginx instance. The nginx server is insturemnted with [nginx-otel module](https://github.com/nginxinc/nginx-otel/tree/main)

Reviewer: ```suggestion
This service provides the images which are used in the frontend. The images are statically hosted on a nginx instance. The nginx server is instrumented with the [nginx-otel module](https://github.com/nginxinc/nginx-otel/tree/main).
```

> This service provides the images which are used in the frontend. The images are statically hosted on a nginx instance. The nginx server is instrumented with the [nginx-otel module](https://github.com/nginxinc/nginx-otel/tree/main).

Reviewer: ```suggestion
This service provides the images which are used in the frontend. The images are statically hosted on a NGINX instance. The NGINX server is instrumented with the [nginx-otel module](https://github.com/nginxinc/nginx-otel/tree/main).
```
